db/fixtures/miq_event_definitions.csv

```diff
@@ -106,7 +106,7 @@ vm_provisioned,VM Provision Complete,Default,vm_process
 vm_start,VM Power On,Default,vm_operational
 request_vm_start,VM Power On Request,Default,vm_operational
 # vm_shutdown,VM Shutdown,Default,vm_operational
-# vm_resume,VM Resume,Default,vm_operational
+vm_resume,VM Resume,Default,vm_operational
```

Comment:
@lfu Are there events from other providers that should be using this vm_resume policy event?